WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -289,9 +289,9 @@ - (void)applicationDidEnterBackground:(UIApplication *)application
         // Synchronize the cleanup call on the main thread in case
         // the task actually finishes at around the same time.
         dispatch_async(dispatch_get_main_queue(), ^{
-            if (_bgTask != UIBackgroundTaskInvalid) {
-                [app endBackgroundTask:_bgTask];
-                _bgTask = UIBackgroundTaskInvalid;
+            if (self->_bgTask != UIBackgroundTaskInvalid) {
```

Comment:
that could probably just be `self.bgTask`, right?

Reply:
oh, there's a bunch more of those. up to you then — my personal style is to just do `self.`, because I have a hard time parsing what `->_` means at a glance, but you decide :)

Reply:
yeah let me fix those, I just used XCode autocorrect for those but I don't like how that looks either.